packages/databricks-vscode/resources/python/notebook.workflow-wrapper.py

```diff
@@ -0,0 +1,12 @@
+#This file is autogenerated by the Databricks Extension for VS Code
+
+dbutils.widgets.text("DATABRICKS_SOURCE_FILE", "")
+dbutils.widgets.text("DATABRICKS_PROJECT_ROOT", "")
+
+if dbutils.widgets.get("DATABRICKS_SOURCE_FILE") != "":
+	import os
+	os.chdir(os.path.dirname(dbutils.widgets.get("DATABRICKS_SOURCE_FILE")))
+
+if dbutils.widgets.get("DATABRICKS_PROJECT_ROOT") != "":
+	import sys
+	sys.path.append(dbutils.widgets.get("DATABRICKS_PROJECT_ROOT"))
```

Comment:
in "file" mode you also append the dirname of the file.

Reply:
The dirname is already set currectly. It is redundant in the file mode. Removing from there.